base/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM openjdk:11.0.1-jdk
 
-# man directory is missing in some base images
+# Man directory is missing in some base images
 # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
 RUN apt-get update \
-    && mkdir -p /usr/share/man/man1 \
+    && mkdir -p /usr/share/man/man1 \ 
```

Comment:
This looks like a typo - the backslash is quoting a space?

Reply:
No. The backslash is not quoting a space, it is extending the command to the next line. I double checked the docker build and this does not affect it. However, I went ahead and removed all trailing whitespace.